public/locales/en.json

```diff
@@ -36,6 +37,7 @@
   "youWillReceive": "You will receive at least",
   "youAreBuying": "You are buying",
   "itWillCost": "It will cost at most",
+  "forAtMost": "for at most",
```

Comment:
can't comment on the actual file, but again let's try to use feather icons wherever possible instead of rolling our own

i'd say `close.png` -> https://feathericons.com/?query=x